docs/en/advanced/Tiered-Locality.md

```diff
@@ -6,59 +6,102 @@ group: Advanced
 priority: 1
 ---
 
-TODO(andrew): Add intro section which talks about why users would want to use this feature (specific use cases)
-TODO(andrew): Include example(s) of how to verify that tiered locality is set up correctly
-TODO(andrew): Focus on setting tiered locality via script, only discuss setting via configuration towards the end
-TODO(andrew): Split into "Basic Setup" vs "Advanced Setup". Most content goes into "Advanced Setup"
-TODO(andrew): Suggest using centralized configuration to set alluxio.locality.order when configuring custom tiers
-
 * Table of Contents
 {:toc}
 
+Alluxio's tiered locality feature enables applications to make intelligent, topology-based decisions
+about which Alluxio workers to read from and write to.
+For an application running on `host1`, reading data from an Alluxio worker on `host1` is more efficient 
+than reading from a worker on `host2`. Similarly, reading from a worker on the same rack or in the same
+datacenter is faster than reading from a worker in a different rack or different datacenter. You can take
+advantage of these various levels of locality by configuring your servers and clients with network 
+topology information.
+
 ## Tiered Identity
 
 In Alluxio, each entity (masters, workers, clients) has a *Tiered Identity*. A tiered
-identity is an address tuple such as `(node=..., rack=...)`. Each pair in the tuple is called
-a *locality tier*. The locality tiers are ordered from most specific to least specific. Alluxio
-uses tiered identities to optimize for locality. For example, when the client wants to read a
-file from the UFS, it will prefer to read through an Alluxio worker on the same node.
-If there is no local worker in the first tier (node), the next tier (rack) will be checked
-to prefer rack-local data transfer. If no workers share a node or rack with the client, an
-arbitrary worker will be chosen.
+identity is an address tuple such as **`(node=node_name, rack=rack_name, datacenter=datacenter_name)`**.
+Each pair in the tuple is called a *locality tier*.
+Alluxio clients favor reading and writing workers that are "more local". For example, if a client has
+identity **`(node=A, rack=rack1, datacenter=dc1)`**, it will prefer to read from a worker at
+**`(node=B, rack=rack1, datacenter=dc1)`** over a worker at **`(node=C, rack=rack2, datacenter=dc1)`**. This keeps the
+read within the same rack instead of going cross-rack.
+
+## Basic Setup
+
+For this example, suppose your Alluxio workers are spread across multiple availability zones within EC2.
+To configure tiered locality, follow these steps:
+
+1. Write a script named `alluxio-locality.sh`:
```

Comment:
This is the first mention of a script. I think there should be an explanation of what the purpose of the script is?

Reply:
added, do you think the new description is adequate?